contracts/Controller.sol

```diff
@@ -650,16 +670,6 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     {
         require(_checkVaultId(_args.owner, _args.vaultId), "Controller: invalid vault id");
 
-        MarginVault.Vault memory vault = getVault(_args.owner, _args.vaultId);
-        if (_isNotEmpty(vault.shortOtokens)) {
-            OtokenInterface otoken = OtokenInterface(vault.shortOtokens[0]);
-
-            require(
-                now < otoken.expiryTimestamp(),
-                "Controller: can not withdraw collateral from a vault with an expired short otoken"
-            );
-        }
-
```

Comment:
Why do we need to remove this and open it up after expiry? :) Just trying to think about what case needs this!

Reply:
I removed this so vault owner can withdraw remaining collateral from vault after liquidation. And added checking margin vault after withdraw collateral action.

Reply:
Added tests for withdrawing more collateral than payout in an expired position, withdrawing some of the collateral and settling should still only give a sum of amounts equal to the payout https://github.com/opynfinance/GammaProtocol/blob/6647517cf4bd30de0b3bf763c70e620a450e223b/test/unit-tests/controller.test.ts#L3640

Reply:
if a vault got liquidated they can still withdraw through add collateral right?

Reply:
would prefer if we kept this check, just want to think through the issues this could cause before opening this up

Reply:
Yeah they can still do that if the short otoken is not expired yet through _withdrawCollateral action, if the short is expired, they settle vault and will get the proceed and the excess amount also.